### ISSUE TITLE
python3 pip doesn't have version 1.0

### ISSUE THREAD

**Issue by AnthonyWC** [jgonggrijp/pip-review#58]
```
pip2 show pip-review 
Name: pip-review
Version: 1.0
Summary: pip-review lets you smoothly manage all available PyPI updates.
Home-page: https://github.com/jgonggrijp/pip-review
Author: Julian Gonggrijp, Vincent Driessen
Author-email: j.gonggrijp@gmail.com
License: BSD
Location: /usr/local/lib/python2.7/dist-packages
Requires: pip, packaging

python3 -m pip show pip-review
Name: pip-review
Version: 0.5.3
Summary: pip-review lets you smoothly manage all available PyPI updates.
Home-page: https://github.com/jgonggrijp/pip-review
Author: Vincent Driessen, Julian Gonggrijp
Author-email: j.gonggrijp@gmail.com
License: BSD
Location: /home/antho/.local/lib/python3.5/site-packages
Requires: packaging, argparse, pip

```

**Comment by xoviat** [comment]
This is not this project's fault. It's either your fault or the fault of someone over at [pypa](https://github.com/pypa).

**Comment by jgonggrijp** [comment]
No, I suspect it is @AnthonyWC's fault for failing to update his python3 `site-packages`. You have separate package directories for each installation of Python and you have to update your packages separately in each. pip-review is no exception to this rule.

See my shell output below. Please reopen and/or comment if you feel I'm in error.

    $ python --version
    Python 3.5.4

    $ pip --version
    pip 9.0.1 from /Users/julian/.env/piprev3/lib/python3.5/site-packages (python 3.5)

    $ pip list -o --format=legacy
    pip-review (0.5.3) - Latest: 1.0 [wheel]
    pip-tools (1.8.1) - Latest: 1.9.0 [wheel]
    py (1.4.33) - Latest: 1.4.34 [wheel]
    pyparsing (2.1.10) - Latest: 2.2.0 [wheel]
    setuptools (33.1.1) - Latest: 36.2.7 [wheel]
    tox (2.6.0) - Latest: 2.7.0 [wheel]

    $ pip-review
    pip-review==1.0 is available (you have 0.5.3)
    pip-tools==1.9.0 is available (you have 1.8.1)
    py==1.4.34 is available (you have 1.4.33)
    pyparsing==2.2.0 is available (you have 2.1.10)
    setuptools==36.2.7 is available (you have 33.1.1)
    tox==2.7.0 is available (you have 2.6.0)

    $ pip show pip-review
    Name: pip-review
    Version: 0.5.3
    Summary: pip-review lets you smoothly manage all available PyPI updates.
    Home-page: https://github.com/jgonggrijp/pip-review
    Author: Vincent Driessen, Julian Gonggrijp
    Author-email: j.gonggrijp@gmail.com
    License: BSD
    Location: /Users/julian/.env/piprev3/lib/python3.5/site-packages/pip_review-0.5.3-py3.5.egg
    Requires: packaging, pip, argparse

    $ pip-review --auto
    Collecting pip-review==1.0
      Downloading pip_review-1.0-py2.py3-none-any.whl
    Requirement already satisfied: packaging in ./.env/piprev3/lib/python3.5/site-packages (from pip-review==1.0)
    Requirement already satisfied: pip in ./.env/piprev3/lib/python3.5/site-packages (from pip-review==1.0)
    Requirement already satisfied: pyparsing in ./.env/piprev3/lib/python3.5/site-packages (from packaging->pip-review==1.0)
    Requirement already satisfied: six in ./.env/piprev3/lib/python3.5/site-packages (from packaging->pip-review==1.0)
    Installing collected packages: pip-review
      Found existing installation: pip-review 0.5.3
        Uninstalling pip-review-0.5.3:
          Successfully uninstalled pip-review-0.5.3
    Successfully installed pip-review-1.0
    Collecting pip-tools==1.9.0
    # and so forth

    $ pip show pip-review
    Name: pip-review
    Version: 1.0
    Summary: pip-review lets you smoothly manage all available PyPI updates.
    Home-page: https://github.com/jgonggrijp/pip-review
    Author: Julian Gonggrijp, Vincent Driessen
    Author-email: j.gonggrijp@gmail.com
    License: BSD
    Location: /Users/julian/.env/piprev3/lib/python3.5/site-packages
    Requires: packaging, pip

**Comment by AnthonyWC** [comment]
Somehow I missed this update but the command needed was:
`python3 -m pip install -U pip-review --user`

Due to the user specific install that I had.
